src/System.Collections.Concurrent/tests/ConcurrentDictionary/ConcurrentDictionaryTests.cs

```diff
@@ -546,25 +546,51 @@ public static void TestDebuggerAttributes()
         }
 
         [Fact]
-        public static void TestConstructor_Negative()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Framework hasn't received the fix for https://github.com/dotnet/corefx/issues/18432 yet.")]
+        public static void TestNullComparer()
```

Comment:
Sure, no problem. I'll get a chance do that tonight or tomorrow morning.
The tests currently prove that it's getting _some_ default comparer because otherwise there would be a NRE. I'm not aware of any other comparer source it could even be defaulting to.